apps/judicial-system/web/src/routes/Court/Indictments/Conclusion/Conclusion.tsx

```diff
@@ -0,0 +1,302 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+import router from 'next/router'
+
+import {
+  Box,
+  Checkbox,
+  Input,
+  InputFileUpload,
+  RadioButton,
+  // toast,
+} from '@island.is/island-ui/core'
+import * as constants from '@island.is/judicial-system/consts'
+import {
+  core,
+  //  errors,
+  titles,
+} from '@island.is/judicial-system-web/messages'
+import {
+  BlueBox,
+  CourtArrangements,
+  CourtCaseInfo,
+  FormContentContainer,
+  FormContext,
+  FormFooter,
+  // Modal,
+  PageHeader,
+  PageLayout,
+  PageTitle,
+  SectionHeading,
+  useCourtArrangements,
+} from '@island.is/judicial-system-web/src/components'
+import {
+  CaseFileCategory,
+  // CaseTransition,
+} from '@island.is/judicial-system-web/src/graphql/schema'
+import { stepValidationsType } from '@island.is/judicial-system-web/src/utils/formHelper'
+import {
+  useCase,
+  useS3Upload,
+  useUploadFiles,
+} from '@island.is/judicial-system-web/src/utils/hooks'
+
+import { strings } from './Conclusion.strings'
+
+type Actions = 'POSTPONE'
+
+interface Postponement {
+  postponedIndefinitely?: boolean
+  reason?: string
+}
+
+const Conclusion: React.FC = () => {
+  const { formatMessage } = useIntl()
+  const { workingCase, setWorkingCase, isLoadingWorkingCase, caseNotFound } =
+    useContext(FormContext)
+  // const [navigateTo, setNavigateTo] = useState<keyof stepValidationsType>()
+
+  const [selectedAction, setSelectedAction] = useState<Actions>()
+  const [postponement, setPostponement] = useState<Postponement>()
+  const {
+    courtDate,
+    handleCourtDateChange,
+    handleCourtRoomChange,
+    sendCourtDateToServer,
+  } = useCourtArrangements(workingCase, setWorkingCase, 'courtDate')
+
+  const {
+    // transitionCase,
```

Comment:
Do we want to keep all this commented out code and merge it to main or just remove it now and then write it in again in the branches that will handle these tasks ?

Reply:
I was wondering myself, but thought it would be useful to keep. There is a modal and and all kinds of stuff in there that we may want to use in future PRs. But on second thought I will remove it and all related code as well. It should be easy to retrieve it again.